server/database/createSeed.js

```diff
@@ -74,7 +74,7 @@ const saveToFile = function(array, filePath) { //array is what is being saved, f
     fd = fs.openSync(filePath, 'as');
```

Comment:
Some good looking optimizations.

Reply:
I wasn't sure what to choose for that one... I think there might be a more efficient version, but I believe that's the flag which specifically is designed to only deal with appends so I assume as you said that it is optimized.